tests/sentry/post_process_forwarder/test_post_process_forwarder.py

```diff
@@ -45,10 +45,9 @@ def kafka_message_payload() -> Any:
 
 class PostProcessForwarderTest(TestCase):
     def _get_producer(self, cluster_name: str) -> Producer:
+        bootstrap_servers = settings.KAFKA_CLUSTERS[cluster_name]["common"]["bootstrap.servers"]  # type: ignore
```

Comment:
""

Reply:
not sure what this comment means, maybe it got resolved?

you could use `get_topic_definition` here

Reply:
@untitaker do you know why this isn't resolving the right cluster?

Reply:
Oh nvm, i see i'm using it wrong